### DISCUSSION TITLE
Old bootloaders

### DISCUSSION THREAD

**Discussion by weavermedia** [SynthstromAudible/DelugeFirmware#154] [Q&A]
Is there any work being done on the Synthstrom side to upgrade old bootloaders?

After the first Q&A with Rohan I thought the only issue with old bootloaders is that they can get stuck with zero byte or over-sized firmware binaries. But [this issue](https://github.com/SynthstromAudible/DelugeFirmware/issues/134) shows that even properly formed firmware can break on these old bootloader versions.

Or am I conflating two completely different things here?

**Comment by sapphire-arches** [comment]
That bug is specific to community firmware. We're somehow generating
binaries incompatible with the older bootloaders. There's instructions
(provided by synthstrom) on the GitHub wiki about how to update your
bootloader but we want to maintain compatibility regardless.

**Comment by weavermedia** [comment]
Aha!, I had no idea a bootloader update process was now available. There's a lot of information coming and it's quite spread out. Now I know to check the wiki too. Thanks! 🙏
